dashboard/app/models/user_permission.rb

```diff
@@ -59,6 +59,11 @@ class UserPermission < ApplicationRecord
 
   after_save :log_permission_save
   before_destroy :log_permission_delete
+  after_create :send_verified_teacher_email, if: proc {permission == AUTHORIZED_TEACHER}
+
+  def send_verified_teacher_email
+    TeacherMailer.verified_teacher_email(user).deliver_now if user&.email&.present?
```

Comment:
tiny tiny nit: could this be `if user&.email` or `if user&.email.present?` instead?

Reply:
I'm with you and slightly prefer the `.present?` way so fine to keep it as is or just remove the extra `&`